RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -136,7 +159,7 @@ def TICL_iterations(process):
       seeding_regions = "ticlSeedingGlobal",
       missing_layers = 3,
       min_clusters_per_ntuplet = 15,
-      min_cos_theta = 0.99 # ~10 degrees
+      min_cos_theta = 0.99, # ~10 degrees
```

Comment:
is the out-in supposed to be enabled here? it is disabled in the TICL_iterations_withReco

Reply:
@slava77 I will drop `TICL_iterations` without Reco in another PR since now we link with tracks and it makes little sense to have a "standalone TICL".